premis_event_service/presentation.py

```diff
@@ -137,12 +142,35 @@ def premisEventXMLgetObject(eventXML):
     the same identifier
     """
 
-    identifierValue = ''
-    for element in eventXML:
-        if element.tag == '{http://www.w3.org/2005/Atom}id':
-            identifierValue = element.text
-    ExistingObject = get_object_or_404(Event, event_identifier=identifierValue)
-    return ExistingObject
+    identifierValue = None
+    # Look for Event ID in entry metadata.
+    try:
+        identifierValue = eventXML.xpath('//id')[0].text
+    except (etree.LxmlError, IndexError):
+        identifierValue = None
```

Comment:
Just seems repetitive to me, since the var was already explicitly initialized with a value of `None`. But it's certainly not hurting anything.